src/pages/monetization/app-review-process/index.mdx

```diff
@@ -10,38 +10,60 @@ desc: "If you're into building LiveChat integrations, but don't know how to go a
+With over <Placeholder id="NUMBER_OF_CUSTOMERS" /> businesses using LiveChat daily, we want to make sure that what we offer in the Marketplace, provides our customers with tangible value and enriches their LiveChat experience. For that reason, all apps in the LiveChat Marketplace must meet our basic criteria for quality and content.
 
-The decision to feature your app on the LiveChat Marketplace depends mainly on technical requirements, but not only. We also assess how your app contributes to the LiveChat ecosystem and community. This document covers most cases, however, some exceptions may be allowed at the reviewer’s discretion.
+This document will provide you with knowledge about the review processes offered by LiveChat and some good practices, guidelines, and rules that may help you to make your app an attractive and efficient product.
 
-# Developer's responsibilities
+# App review services
 
-As a developer, you need to know how we go about reviewing apps and how to make your app go through the process with success.
+To ensure that your app meets the basic criteria to be published on our Marketplace and provides your customers with high-quality service, there are two app review services offered by LiveChat:
 
-Each app is thoroughly checked before releasing it to the LiveChat Marketplace. It involves a team of three or more specialists.
-Since we aim to offer top-quality products, we've prepared the list of responsibilites for developers and app owners. We highly encourage you to familiarize yourself with all the aspects.
+## Basic app review
+
+Mandatory for every app before you can publish it on the LiveChat Marketplace. The basic review is free of charge. During this process, our experts make sure if:
+
+- The most crucial functions of your application work and if the app has a business value.
+- The app doesn't use [derogatory language](#restricted-content).
+- LiveChat/HelpDesk names included in the app’s name (if applicable) are written correctly.
+- Your application has a tutorial, icon, and at least one screenshot. Read more about Marketplace listing content [here](#app-details).
+
+## Premium app review
+
+We would like to offer you an additional paid review of your app. It includes detailed technical, design, and copy reviews with feedback or suggestions on how to improve your product. Premium app review service involves a team of specialists making sure your app gives your customers the best experience.
+
+You can buy our service directly on our Marketplace. However, if you would like to compare both services first, see App Review Offer.
+
+# Best practices
+
+Since we aim to offer a top-quality experience and you would like to provide your future customers with top-quality products, we've prepared the list of Best practices for developers and app owners. We highly encourage you to familiarize yourself with all the aspects.
```

Comment:
```suggestion
Since we aim to offer a top-quality experience and you would like to provide your future customers with top-quality products, we've prepared a list of best practices for developers and app owners. We highly encourage you to familiarize yourself with all the aspects.
```